components/post_view/post_list_virtualized/post_list_virtualized.jsx

```diff
@@ -118,18 +132,20 @@ class PostList extends React.PureComponent {
         this.state = {
             isScrolling: false,
             isMobile,
-            atBottom: false,
+            atBottom: true,
             lastViewedBottom: Date.now(),
             postListIds: [channelIntroMessage],
             topPostId: '',
             postMenuOpened: false,
             dynamicListStyle: {
                 willChange: 'transform',
             },
+            unreadCountInChannel: props.unreadCountInChannel,
         };
 
         this.listRef = React.createRef();
         this.postListRef = React.createRef();
+        this.forceUnreadEvenAtBottom = true;
```

Comment:
What does this do?

Reply:
I believe this is to prevent the toast from fading when there is a timeout sequence.

Reply:
Normally toasts disappear when at bottom quickly but if when landing in on a channel with unreads and is at bottom(less than a page of unreads will keep the scroll position to bottom). This forces the toast to stay for a little while.